web_monitoring/db.py

```diff
@@ -44,4 +44,4 @@ def query_import_status(import_id):
     url = WEB_MONITORING_SHOW_IMPORT_API.format(db_url=settings['db_url'],
                                                 import_id=import_id)
     return requests.get(url,
-                        auth=(settings['db_email'], settings['db_password']))
+                        auth=(settings['db_email'], settings['db_password']))
```

Comment:
This is a minor nit, but can you make sure to end files with a blank line to make Unix tools more happy?